packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/entities/Annotation.java

```diff
@@ -55,6 +55,9 @@ public class Annotation extends PanacheEntityBase implements Serializable {
 	@JsonProperty("artifact_type")
 	private String artifactType;
 
+	@Column(nullable = true)
+	private String section;
```

Comment:
What is this field meant to convey? Where the annotation points to? 

I wonder if we want to have a a more generic data representation now to avoid changing this a few more times in the future? For example one might imagine you may have a starting-characer-number, ending-character-number kind of thing. 

Any thoughts?

Reply:
They are predefined sections. See https://github.com/DARPA-ASKEM/Terarium/issues/807